GPCS4/SceModules/SceLibkernel/sce_kernel_file.cpp

```diff
@@ -66,6 +67,24 @@ inline bool getDirName(DIR* dir, char* dirname, int len)
 
 #endif  //GPCS4_WINDOWS
 
+
+int PS4API scek__write(int fd, const void* buf, size_t size)
+{
```

Comment:
forget `LOG_SCE_TRACE` ?

Reply:
hm?
![image](https://user-images.githubusercontent.com/6632271/69903203-449ba400-139f-11ea-9581-334b639d68e1.png)

Reply:
Still need a `LOG_SCE_TRACE` to indicate `scek__write` has been called.

Reply:
oh I see